scripts/pre_leap.sh

```diff
@@ -19,6 +19,10 @@
 set -e -u
 set -o pipefail
 
+# Preserve Elasticsearch data in the leap
+export UPGRADE_ELASTICSEARCH=${UPGRADE_ELASTICSEARCH:-"yes"}
+export CONTAINERS_TO_DESTROY=${CONTAINERS_TO_DESTROY:-""all_containers:!galera_all:!neutron_agent:!ceph_all:!rsyslog_all:!elasticsearch_all""}
```

Comment:
No need to have the `""` around our default. This should likely just be single quoted. 

``` shell
export CONTAINERS_TO_DESTROY=${CONTAINERS_TO_DESTROY:-'all_containers:!galera_all:!neutron_agent:!ceph_all:!rsyslog_all:!elasticsearch_all'}
```

Additionally I'm not sure we need to have this be configurable? We know these are the container groups and types we want to keep so it may be better for us to just set it and forget it.

``` shell
export CONTAINERS_TO_DESTROY='all_containers:!galera_all:!neutron_agent:!ceph_all:!rsyslog_all:!elasticsearch_all'
```

Reply:
Thought we might have the situation where the customer doesn't want to keep the data and there may not be a point in keeping it.  In this case, they'd have the option to override.

Reply:
Double-quote sometimes lose function and failed to exporting value to 'CONTAINERS_TO_DESTROY', so double them to make sure it will work. I'm testing single-quote and double double-quote now.